evap/evaluation/tests/tools.py

```diff
@@ -341,6 +346,91 @@ def setUpClass(cls) -> None:
         cls.selenium.set_window_size(*cls.window_size)
 
 
+@tag("vrt")
+class VisualRegressionTestCase(LiveServerTest):
+    window_size = (1920, 1080)
+    _http_timeout_seconds = 3
+
+    def setUp(self) -> None:
+        with connection.cursor() as cursor:
+            # reset userprofile id sequence to keep test reproducible
+            cursor.execute("ALTER SEQUENCE evaluation_userprofile_id_seq restart")
+
+        super().setUp()
+
+    @property
+    def viewport(self):
+        return f"{self.window_size[0]}x{self.window_size[1]}"
+
+    def trigger_screenshot(self, name: str):
+        full_name = self.__class__.__name__ + "_" + name
+
+        config = {
+            "apiUrl": os.environ.get("VRT_APIURL"),
```

Comment:
intuitively this should happen in `VisualRegressionTestCase.__init__`; any reason against it?